src/charts/utilization/utilization-chart.html

```diff
@@ -1,7 +1,7 @@
 <div class="utilization-chart-pf">
-    <span class="title">{{config.title}}</span>
+    <span class="h3">{{config.title}}</span>
```

Comment:
It's arguable that's a bug in patternfly.  I would expect `.content .h3` to be equivalent to `.content h3`.  What if we override the font-size and line-height properties using `.utilization-chart-pf`?